Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/LoadingProgressManager.cs

```diff
@@ -106,8 +111,15 @@ void RemoveTracker(ulong clientId)
                 var tracker = ProgressTrackers[clientId];
                 ProgressTrackers.Remove(clientId);
                 tracker.NetworkObject.Despawn();
-                UpdateTrackersClientRpc();
+                // This makes sure that clients received the Despawn message before the RPC.
```

Comment:
This only makes sure RPCs are sent in different messages. They could be received in the same update client side.
Plus if you're sending messages reliable ordered, they should keep the same order client side.
What's the issue you saw here?

Reply:
Clients would be executing this RPC, but they wouldn't have despawned the networkObjects before executing it.
But good point, there is no guarantee of during which frame they are received client-side. I have some alternative solutions in mind that would be better